apps/financial-aid/backend/src/app/modules/application/application.service.ts

```diff
@@ -154,4 +169,14 @@ export class ApplicationService {
 
     return { numberOfAffectedRows, updatedApplication }
   }
+
+  async createEvent(
+    applicationEvent: CreateApplicationEventDto,
+  ): Promise<ApplicationEventModel> {
+    console.log('Creating a new application event')
+
+    const appEvent = await this.applicationEventService.create(applicationEvent)
```

Comment:
Should the `ApplicationController` talk to this service instead of the `ApplicationServicer` talking to it directly?